src/py/mains/run.py

```diff
@@ -38,7 +38,9 @@ def run_script(job_data, job_type = "SGE" ):
         job_name = job_data["job_name"]
         cwd = job_data["cwd"]
         sge_option = job_data["sge_option"]
-        fc_run_logger.info( "submitting %s for SGE, start job: %s " % (script_fn, job_name) )
+        with open(script_fn, 'r') as original: data = original.read()
+	with open(script_fn, 'w') as modified: modified.write("#!/bin/sh" + "\n" + data)
```

Comment:
Interesting. I wonder whether we could have used 'bash {script}' on the sbatch line.

Anyway, I'm fine with this -- aside from the tab character.